### PR TITLE
FZ83P-11

### PR DESCRIPTION
**Pull request by bartzaw**
I've added flex to add-info section along with mobile-first model. Added section container to better position all elements.

### CODE REVIEW

**Review by sebastian-kreft — CHANGES_REQUESTED** [review]
How is this PR different from this one: https://github.com/infoshareacademy/jfdz8-3-promile/pull/11

**Review by icelandico — APPROVED** [review]
Looks fine for me.

**Comment by bartzaw** [comment]
hi @sebastian-kreft - this one relates to additional-info section, the one you've mentioned relates to features section. By mistake I've amended SOME of add-info styles in previous pull request, but this one you are reviewing now is suppose to change whole section to act as I'd like it to. I know why there's confusion around this one - I pulled all changes from another branch - FZ83P-6, perhaps it wasn't necessary but I wanted to see how all sections will react to my changes.

**Review by Decomplosed — APPROVED** [review]
All good!

File: assets/style/style.css [hunk 1]
```diff
@@ -241,8 +241,8 @@ h2 {
 }
 
 .section-features__header, .add-info__header {
-    font-size: 24px;
     padding: 2%;
+    font-size: 2rem;
```
Comment (sebastian-kreft) [inline assets/style/style.css:245]: I hope you know exactly how `rem` and `em` works and why you chose it :)

**Review by sebastian-kreft — APPROVED** [review]
It looks good. However, please be sure about using relative values for font sizes.

**Pull request merged**